src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -860,13 +926,13 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
     }
   }
 
-  private clear() {
+  private dismiss() {
```

Comment:
I don't think `dismiss` is descriptive enough.

`dismissSearchAndRemoveMoreLessAndClear()` would be the actual description of what it does, and, well, it's kinda telling us that the method is probably doing too much here.

We might wanna split it in shorter and more concise function.